subsys/bluetooth/controller/Kconfig

```diff
@@ -584,5 +584,13 @@ config BT_CTLR_SDC_CS_COUNT
 	help
 	  Set the number of concurrent connections that can support Channel Sounding procedure.
 
+config BT_CTLR_SDC_LE_POWER_CLASS_1
+	bool "Device supports transmitting at LE Power Class 1 level"
+	default n
```

Comment:
Is this needed?
https://docs.zephyrproject.org/latest/build/kconfig/tips.html#redundant-defaults